metadata_annotation_farland.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<p>Sick</p><p> </p><p>Could we get a one-liner for what kind of annotation each resource provides?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>